doc/languages-frameworks/javascript.section.md

```diff
@@ -342,8 +342,8 @@ stdenv.mkDerivation (finalAttrs: {
 })
 ```
 
-NOTE: It is highly recommended to use a pinned version of pnpm (i.e. `pnpm_8` or `pnpm_9`), to increase future reproducibility. It might also be required to use an older version, if the package needs support for a certain lock file version.
-
+It is highly recommended to use a pinned version of pnpm (i.e. `pnpm_8` or `pnpm_9`), to increase future reproducibility. It might also be required to use an older version, if the package needs support for a certain lock file version. The fetchDeps hook will abort, if the used pnpm version is to old.
```

Comment:
Typo
```suggestion
It is highly recommended to use a pinned version of pnpm (i.e. `pnpm_8` or `pnpm_9`), to increase future reproducibility. It might also be required to use an older version, if the package needs support for a certain lock file version. The fetchDeps hook will abort if the used pnpm version is too old.
```